ui/src/app/pages/workspace/create-billing-account-modal.tsx

```diff
@@ -306,7 +304,7 @@ export const CreateBillingAccountModal = ({onClose}: Props) => {
               </FlexColumn>
             </FlexRow>
             <FlexRow style={{marginTop: '10px'}}>
-              <CheckBox style={{margin: '15px',  width: '17px', height: '17px', flexShrink: 0, maxWidth: '17px'}}
+              <CheckBox style={styles.radioButton}
                         checked={nihFunded === true}
                         onChange={() => setNihFunded(false)}/>
```

Comment:
actually, this looks like a bug. It only goes in one direction, and it's a checkbox, needs to consider the checked value